modules/dsp_genieeBidAdapter.js

```diff
@@ -42,21 +42,21 @@ export const spec = {
   code: BIDDER_CODE,
   supportedMediaTypes: [BANNER],
   /**
-     * Determines whether or not the given bid request is valid.
-     *
-     * @param {BidRequest} - The bid params to validate.
-     * @return boolean True if this is a valid bid, and false otherwise.
-     */
+ * Determines whether or not the given bid request is valid.
+ *
+ * @param {BidRequest} - The bid params to validate.
+ * @return boolean True if this is a valid bid, and false otherwise.
+ */
```

Comment:
```suggestion
     * Determines whether or not the given bid request is valid.
     *
     * @param {BidRequest} - The bid params to validate.
     * @return boolean True if this is a valid bid, and false otherwise.
     */
```